README.md

```diff
@@ -70,6 +70,14 @@ is useful for this; for more information please see the
 [`partner-charts-ci` documentation](https://github.com/rancher/partner-charts-ci?tab=readme-ov-file#4-run-partner-charts-ci-update).
 
 
```

Comment:
This empty line is not needed.

Reply:
The readme currently has two empty lines after each section. I like doing it this way because it makes the file easier to read in a text editor. I can change this if you want, but it doesn't have any effect on the HTML version of the file.